wcivf/apps/hustings/models.py

```diff
@@ -1,14 +1,31 @@
 """
 Models for Hustings
 """
-import datetime
+from django.utils import timezone
 import hashlib
 
 from django.db import models
 
 from elections.models import PostElection
 
 
+class HustingQueryset(models.QuerySet):
+    def future(self):
+        """
+        Returns QuerySet of objects in the future or today
+        """
+        return self.filter(starts__date__gte=timezone.now().date())
+
+    def displayable(self):
+        """
+        Excludes objects in the past unless we have a postevent url for them
```

Comment:
I don't think this does what the comment says? This will always exclude events in the past, I think? Might need a `Q() | Q()` query?